telescope/src/main/java/com/mattprecious/telescope/TelescopeLayout.java

```diff
@@ -59,7 +59,7 @@
  */
 public class TelescopeLayout extends FrameLayout {
   private static final String TAG = "Telescope";
-  static final SimpleDateFormat SCREENSHOT_FILE_FORMAT =
```

Comment:
Hmm weird. We just suppressed this internally with a comment that it's accesses were thread-confined. If only the `@MainThread` annotation allowed field targets... http://b.android.com/208133